meson_options.txt

```diff
@@ -47,3 +47,6 @@ option('curl', type : 'boolean', value : true, description : 'enable libcurl sup
 option('udevdir', type: 'string', value: '', description: 'Directory for udev rules')
 option('efi_os_dir', type: 'string', description : 'the hardcoded name of OS directory in ESP, e.g. fedora')
 option('efi_binary', type: 'boolean', value : true, description : 'generate uefi binary if missing')
+option('metainfo', type: 'boolean', value : true, description : 'install the project metainfo.xml information')
+option('fish_completion', type: 'boolean', value : true, description : 'install the fish integration files')
+option('offline', type: 'boolean', value : true, description : 'enable installing offline firmware')
```

Comment:
This wording will be very confusing. Offline doesn't have the obvious meaning here

Reply:
`enable installing firmware using a pre-boot systemd target` perhaps?

Reply:
Yes

Reply:
Fixed.